### PR TITLE
fix non-arc, cannot create __weak reference issue

### PR DESCRIPTION
**Pull request by sanhue0302**
for xcode 7, non-arc, cannot create __weak reference
so just remove __weak from GDataProgressMonitorInputStream.h

### CODE REVIEW

**Comment by sanhue0302** [comment]
I signed it

**Comment by sanhue0302** [comment]
i found that there is no place use "GDataProgressMonitorInputStream.h" on current master commit
so can we simply remove this file?

File: Source/Networking/GDataProgressMonitorInputStream.h [hunk 1]
```diff
@@ -47,11 +47,11 @@
   unsigned long long dataSize_;     // size of data in the source
   unsigned long long numBytesRead_; // bytes read from the input stream so far
 
-  __weak id monitorDelegate_;
+  id monitorDelegate_;
```
Comment (thomasvl) [inline Source/Networking/GDataProgressMonitorInputStream.h]: Can you throw a comment on here to still capture that it is weak?

  id monitorDelegate_;  // weak

File: Source/Networking/GDataProgressMonitorInputStream.h [hunk 2]
```diff
@@ -47,11 +47,11 @@
...
   SEL monitorSelector_;
   SEL readSelector_;
 
-  __weak id monitorSource_;
+  id monitorSource_;
```
Comment (thomasvl) [inline Source/Networking/GDataProgressMonitorInputStream.h]: same

**Pull request closed without merging**